extensions/guacamole-auth-duo/pom.xml

```diff
@@ -209,19 +209,11 @@
 
     <dependencies>
 
-        <!-- Guacamole Java API -->
-        <dependency>
```

Comment:
No, makes sense to me - it's just good to have some information on why these are being removed in a PR that's about bumping version numbers. 😃